monitor/.env.example

```diff
@@ -4,8 +4,18 @@ HOME_BRIDGE_ADDRESS=0xABb4C1399DcC28FBa3Beb76CAE2b50Be3e087353
 FOREIGN_BRIDGE_ADDRESS=0xE405F6872cE38a7a4Ff63DcF946236D458c2ca3a
 HOME_DEPLOYMENT_BLOCK=0
 FOREIGN_DEPLOYMENT_BLOCK=0
-GAS_PRICE_SPEED_TYPE=standard
-GAS_LIMIT=300000
-GAS_PRICE_FALLBACK=21
+
+HOME_GAS_LIMIT=300000
+HOME_GAS_PRICE_ORACLE_URL=https://gasprice.poa.network/
+HOME_GAS_PRICE_SPEED_TYPE=standard
+HOME_GAS_PRICE_FALLBACK=1000000000
```

Comment:
Does the monitor wait the gas price fallback value in gwei?

Reply:
Good catch! All parameters for gas price in UI, Oracle and contract deployment are expected to be in `wei`, monitor is the only one that expect the gas price fallback value in `gwei`. Does it makes sense to define it here in `wei` too for consistency and to avoid confusions? Actually the changes introduced in this PR for monitor expect the gas price fallback to be in `wei`. If we want to follow this way, we only will need to update the env vars in `deployment` to also be in `wei`

Reply:
> Does it makes sense to define it here in wei too for consistency and to avoid confusions?

I believe it could be done when functionality to work with gas price oracles is part of `commons`.

Reply:
> If we want to follow this way, we only will need to update the env vars in deployment to also be in wei

What minimum amount of changes needs to be done to address this issue within this PR?

Reply:
Well on this PR `HOME_GAS_PRICE_FALLBACK` and `FOREIGN_GAS_PRICE_FALLBACK` were introduced and expressed in `wei` for monitor and the code also expect them to be in `wei`. So the only pending changes would be to update `MONITOR_HOME_GAS_PRICE_FALLBACK` and
`MONITOR_FOREIGN_GAS_PRICE_FALLBACK` to be in `wei` instead of `gwei` in files from `deployment/group_vars/`

Reply:
Here I applied the changes, also related to other comments https://github.com/poanetwork/tokenbridge/pull/142/commits/7d940b33bd27c1f8d7c2d1a933a8c4b58725be89

Reply:
Ok. Now I see what you meant. Thanks!